packages/flutter_tools/lib/src/build_system/targets/assets.dart

```diff
@@ -10,81 +10,48 @@ import '../../devfs.dart';
-
-  @override
-  List<File> inputs(Environment environment) {
-    final AssetBundle assetBundle = AssetBundleFactory.instance.createBundle();
-    assetBundle.build(
-      manifestPath: environment.projectDir.childFile('pubspec.yaml').path,
-      packagesPath: environment.projectDir.childFile('.packages').path,
-    );
-    // Filter the file type to remove the files that are generated by this
-    // command as inputs.
-    final List<File> results = <File>[];
-    final Iterable<DevFSFileContent> files = assetBundle.entries.values.whereType<DevFSFileContent>();
-    for (DevFSFileContent devFsContent in files) {
-      results.add(fs.file(devFsContent.file.path));
-    }
-    return results;
-  }
-
-  @override
-  List<File> outputs(Environment environment) {
-    final AssetBundle assetBundle = AssetBundleFactory.instance.createBundle();
-    assetBundle.build(
-      manifestPath: environment.projectDir.childFile('pubspec.yaml').path,
-      packagesPath: environment.projectDir.childFile('.packages').path,
-    );
-    final List<File> results = <File>[];
-    for (String key in assetBundle.entries.keys) {
-      final File file = fs.file(fs.path.join(environment.buildDir.path, 'flutter_assets', key));
-      results.add(file);
-    }
-    return results;
-  }
-}
-
-/// A specific asset behavior for building bundles.
-class AssetOutputBehavior extends SourceBehavior {
-  const AssetOutputBehavior([this._pathSuffix = '']);
-
-  final String _pathSuffix;
-
-  @override
-  List<File> inputs(Environment environment) {
-    final AssetBundle assetBundle = AssetBundleFactory.instance.createBundle();
-    assetBundle.build(
-      manifestPath: environment.projectDir.childFile('pubspec.yaml').path,
-      packagesPath: environment.projectDir.childFile('.packages').path,
-    );
-    // Filter the file type to remove the files that are generated by this
-    // command as inputs.
-    final List<File> results = <File>[];
-    final Iterable<DevFSFileContent> files = assetBundle.entries.values.whereType<DevFSFileContent>();
-    for (DevFSFileContent devFsContent in files) {
-      results.add(fs.file(fs.path.join(_pathSuffix, devFsContent.file.path)));
-    }
-    return results;
-  }
-
-  @override
-  List<File> outputs(Environment environment) {
-    final AssetBundle assetBundle = AssetBundleFactory.instance.createBundle();
-    assetBundle.build(
-      manifestPath: environment.projectDir.childFile('pubspec.yaml').path,
-      packagesPath: environment.projectDir.childFile('.packages').path,
-    );
-    final List<File> results = <File>[];
-    for (String key in assetBundle.entries.keys) {
-      final File file = fs.file(fs.path.join(environment.outputDir.path, _pathSuffix, key));
-      results.add(file);
-    }
-    return results;
-  }
+import '../depfile.dart';
+
+/// A helper function to copy an asset bundle into an [environment]'s output
+/// directory.
+///
+/// [pathSuffix] may be optionally provided to add additional paths to the
```

Comment:
It looks like this parameter no longer eixsts.

Reply:
Removed